src/lib/utils/preconditions/PreconditionContainerSingle.ts

```diff
@@ -64,9 +64,9 @@ export class PreconditionContainerSingle implements IPreconditionContainer {
 	 * @param message The message that ran this precondition.
 	 * @param command The command the message invoked.
 	 */
-	public run(message: Message, command: Command) {
+	public run(message: Message, command: Command, context: PreconditionContext) {
```

Comment:
Isn't this added required parameter a breaking change? Please document it if so.